utils/custom.js

```diff
@@ -141,6 +154,7 @@ const handleAnonymousData = async (data, req, postAuthorId, myAnonymousId, anony
   }
   return {
     ...data,
+    karmaScores: roundingKarmaScore(user?.karma_score || 0),
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Again, add a type check for `user.karma_score` before passing it to `roundingKarmaScore`.

```diff
- karmaScores: roundingKarmaScore(user?.karma_score || 0),
+ karmaScores: typeof user?.karma_score === 'number' ? roundingKarmaScore(user.karma_score) : 0,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
already handle in the function